### DISCUSSION TITLE
miscellany/life/20240721-home-life

### DISCUSSION THREAD

**Discussion by giscus** [kands-code/kands-code.github.io#82] [General]
# miscellany/life/20240721-home-life

找到属于自己的光

https://blogs.qarks.top/miscellany/life/20240721-home-life.html

**Comment by kands-code** [comment]
最近要慢慢恢复博客更新了，重点应该是代码和数学，偶尔可能吐槽一下生活，感觉这种形式的日记也是必要的，同时也算是社会纪实吧（大概）
